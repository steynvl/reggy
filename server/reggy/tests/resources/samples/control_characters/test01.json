{
  "sampleStringsInfo": [
    [
      {
        "markerType": "Control characters",
        "markedStrings": [
          "asas"
        ],
        "markerInfo": {
          "nul": true,
          "startOfHeading": true,
          "startOfText": true,
          "endOfText": true,
          "endOfTransmission": true,
          "enquiry": true,
          "acknowledge": true,
          "bell": true,
          "backspace": true,
          "horizontalTab": true,
          "newLine": true,
          "verticalTab": true,
          "formFeed": true,
          "carriageReturn": true,
          "shiftOut": true,
          "shiftIn": true,
          "dataLinkEscape": true,
          "deviceControlOne": true,
          "deviceControlTwo": true,
          "deviceControlThree": true,
          "deviceControlFour": true,
          "negativeAcknowledge": true,
          "synchronousIdle": true,
          "endOfTransBlock": true,
          "cancel": true,
          "endOfMedium": true,
          "substitute": true,
          "escape": true,
          "fileSeparator": true,
          "groupSeparator": true,
          "recordSeparator": true,
          "unitSeparator": true,
          "matchAllExceptSelectedOnes": false
        },
        "repeatInfo": {
          "repeat": "1"
        }
      }
    ]
  ],
  "generalRegexInfo": {
    "startRegexMatchAt": "Anywhere",
    "endRegexMatchAt": "Anywhere",
    "regexTarget": "Java"
  },
  "generateMethod": "samplesAndSemantics"
}
