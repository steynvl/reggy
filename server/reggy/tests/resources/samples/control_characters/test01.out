Java
Pattern regex = Pattern.compile("[\\x00-\\x1F]");
[\\x00-\\x1F]
Perl
my $regex = /[\x00-\x1F]/;
[\x00-\x1F]
POSIX
[\x00-\x1F]
[\x00-\x1F]
Python
regex = re.compile(r'[\x00-\x1F]')
[\x00-\x1F]
JavaScript
const regex = /[\x00-\x1F]/;
[\x00-\x1F]
PHP
$regex = '/[\x00-\x1F]/';
[\x00-\x1F]
Golang
regex, _ := regexp.Compile("[\\x00-\\x1F]")
[\\x00-\\x1F]
Rust
let re = Regex::new(r"[\x00-\x1F]").unwrap();
[\x00-\x1F]
C#
var re = new Regex(@"[\x00-\x1F]");
[\x00-\x1F]
Scala
val re = "[\\x00-\\x1F]".r
[\\x00-\\x1F]
Kotlin
val regex = Regex("[\\x00-\\x1F]")
[\\x00-\\x1F]